examples/java/src/test/java/dev/selenium/elements/InteractionTest.java

```diff
@@ -1,5 +1,5 @@
 package dev.selenium.elements;
-
+import dev.selenium.BaseTest;
```

Comment:
Adding this extra line will displace the docs that refer to this code file. If we do not need to import the `BaseTest` class, can we remove it?

Reply:
removed. let me know if anything else required.

Reply:
Thank you!

Reply:
:) thank you Diego.